src/nl/hannahsten/texifyidea/lang/commands/RequiredFileArgument.kt

```diff
@@ -74,6 +74,6 @@ open class RequiredFileArgument(name: String?, open val isAbsolutePathSupported:
     }
 
     override fun matchesExtension(extension: String): Boolean {
-        return supportedExtensions.contains(extension)
+        return supportedExtensions.contains(extension.lowercase(Locale.getDefault()))
```

Comment:
We don't need this anymore right?

Reply:
its actually unused as far as i can tell. putting a breakpoint there never trips